jetty-util/src/main/java/org/eclipse/jetty/util/thread/strategy/AdaptiveExecutionStrategy.java

```diff
@@ -462,7 +465,10 @@ private void execute(Runnable task)
     {
         try
         {
-            _executor.execute(task);
+            if (isUseVirtualThreads())
```

Comment:
Should we make this method protected so that if somebody wants to be more exotic in their logic they can extend?

Reply:
It's already `public`.